src/models/ScheduleModel.ts

```diff
@@ -1,8 +1,12 @@
+import { ScheduleListItem } from "../components/MainSettingsScreen/ScheduleListItem";
 import { AlarmModel } from "./AlarmModel";
 
 export interface ScheduleModel {
     alarms: AlarmModel[];
     enabled: boolean;
     key: number;
     name: string;
+
+    // DO NOT SERIALIZE ANYTHING BELOW THIS LINE
+    listItemRef?: ScheduleListItem;
```

Comment:
Assume this is only temporary until we implement actual models, yeah?

Reply:
Correct assumption

Reply:
It's horrible, awful code that will need to be cleaned up next sprint. Sorry 👎

Reply:
Okay! Just wanted to make sure! ^^